Dockerfile.nightly

```diff
@@ -0,0 +1,56 @@
+# syntax=docker/dockerfile-upstream:experimental
+
+FROM ubuntu:18.04 as build
+
+RUN apt-get update -qq && apt-get install -y \
+    git \
+    cmake \
+    g++ \
+    pkg-config \
+    libssl-dev \
+    curl \
+    llvm \
+    clang \
+    && rm -rf /var/lib/apt/lists/*
+
+COPY ./rust-toolchain /tmp/rust-toolchain
+
+ENV RUSTUP_HOME=/usr/local/rustup \
+    CARGO_HOME=/usr/local/cargo \
+    PATH=/usr/local/cargo/bin:$PATH
+
+RUN curl https://sh.rustup.rs -sSf | \
+    sh -s -- -y --no-modify-path --default-toolchain "$(cat /tmp/rust-toolchain)"
+
+VOLUME [ /near ]
+WORKDIR /near
+COPY . .
+
+ENV CARGO_TARGET_DIR=/tmp/target
+ENV RUSTC_FLAGS='-C target-cpu=x86-64'
+ENV PORTABLE=ON
+RUN --mount=type=cache,target=/tmp/target \
+    --mount=type=cache,target=/usr/local/cargo/git \
+    --mount=type=cache,target=/usr/local/cargo/registry \
+    cargo build -p neard --release --features nightly_protocol --features nightly_protocol_features && \
+    cargo build -p keypair-generator --release --features nightly_protocol --features nightly_protocol_features && \
```

Comment:
we don't really need to build `keypair-generator` and `genesis-csv-to-json`. They are no longer used

Reply:
Can I semd another PR which deletes that code totally? As well we build it for standard releases as well, so probably at least dete it there, but if we dont use it maybe just totally delete it?